docs/samples/Microsoft.ML.Samples/Dynamic/ValueMapping.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.Data.DataView;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingExample
+    {
+        class SampleInfertDataWithFeatures
+        {
+            public float Age = 0;
+            public string Education = default;
+            public string EducationCategory = default;
+        }
+
+        /// This example demonstrates the use of the ValueMappingEstimator by mapping string-to-string values. This is useful
+        /// to map strings to a grouping. In this example, the Education data maps to the groups Undergraduate and Postgraduate:
```

Comment:
>grouping [](start = 32, length = 8)

shall we use the term 'key'

Reply:
are you wanting it to say "map keys to a grouping"? 

---
In reply to: [252890528](https://github.com/dotnet/machinelearning/pull/2222#discussion_r252890528) [](ancestors = 252890528)